content/guides/models/app/features/panels/_index.md

```diff
@@ -125,6 +172,23 @@ To add a section, click **Add section** after the last section.
 
 To add a new section before or after an existing section, you can instead click the section's action `...` menu, then click **New section below** or **New section above**.
 
+
+### Manage a section's panels
+If a section has a large number of panels, by default they are paginated.
+
+- If a section uses the **Standard grid**, you can customize the number of panels to show:
```

Comment:
I'm looking at the UI and I don't see a UI element called **Standard grid** in a project workspace. Is it a UI element or a description of a type of panel layout?

If it's not a UI element, suggest replacing with italicized  and adding a definition. Same applies to Custom grid.

Reply:
Done, PTAL